flow-typed/npm/jest_v27.x.x.js

```diff
@@ -15,204 +15,174 @@
 // The `void` returns are technically wrong. These matchers should be returning
 // {|pass: boolean, message:()=>string|}. See: https://jestjs.io/docs/expect
 export type KACustomMatcherTypes = {|
-  toHaveMarkedConversion: (
-      conversion?: string,
-      conversionExtra?: any,
-      learningTimeInfo?: $FlowFixMe //PartialLearningTimeInfo,
-  ) => void,
-  toMatchSpecificSnapshot: (snapshotName: string) => void,
-  toHaveBeenAnsweredCorrectly(): void,
-  toHaveBeenAnsweredIncorrectly(): void,
-  toHaveInvalidInput(expectedMessage?: string): void,
-  // [Perseus-specific] Asserts that the given DOM element is somewhere
-  // within a highlighted widget
-  toBeHighlighted(): void,
-
-  // KAS specific matchers
-  toBeExpr(string): void,
-  toBeSimplified(options?: $FlowFixMe): void,
-  toCollectAs(string): void,
-  toCollectAsRepr(string): void,
-  toCollectAsTex(string, options?: $FlowFixMe): void,
-  toCompileAs(number, vars?: {[string]: string | number | ((number) => number)}): void,
-  toEqualExpr(string): void,
-  toEqualExprAndForm(string): void,
-  toEvaluateAs(number, vars?: {[string]: string | number}, functions?: $ReadOnlyArray<string>): void,
-  toExpandAs(string): void,
-  toExpandAsRepr(string): void,
-  toExpandAsTex(string): void,
-  toFactorAs(string): void,
-  toFactorAsAsTex(string): void,
-  toFactorAsRepr(string): void,
-  toHaveConsts($ReadOnlyArray<string>): void,
-  toHaveEqualUnits(string): void,
-  toHaveGCD(string): void,
-  toHaveMagnitude(number): void,
-  toHaveNorm(string): void,
-  toHaveStripNorm(string): void,
-  toHaveTheSameForm(string): void,
-  toHaveUnitVariable($FlowFixMe): void,
-  toParseAs(string, options?: $FlowFixMe): void,
-  toParseUnitsAsEqual(string): void,
-  toParseWithStructure(string, options?: $FlowFixMe): void,
-  toRenderTex(string, options?: $FlowFixMe): void,
-  toRenderTexOpt(string, ...optlist: $ReadOnlyArray<$FlowFixMe>): void,
-  toSimplifyAs(string): void,
```

Comment:
**issue:** the removal of these is causing Flow errors.

Reply:
🤦‍♂️  Ah! Thanks for catching that! I just copied this from webapp directly and forgot we have those KAS extensions.  It'd sure be nice if we could split these into separate files.